benchmarks/CMakeLists.txt

```diff
@@ -14,6 +14,14 @@ set_target_properties(
     CXX_STANDARD_REQUIRED ON
 )
 
+target_include_directories(
```

Comment:
You shouldn't need this since the engine includes them as `PUBLIC`.

Reply:
Done.

Reply:
Still here?

Reply:
We can leave it out.